src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const newNumberOne = Big(numberOne);
+  const newNumberTwo = Big(numberTwo);
+  let result;
+  switch (operation) {
+    case '+':
+      result = newNumberOne.plus(newNumberTwo);
+      break;
+    case '-':
+      result = newNumberOne.minus(newNumberTwo);
+      break;
+    case 'X':
+      result = newNumberOne.times(newNumberTwo);
+      break;
+    case '÷':
+      result = numberTwo === 0 ? 'ERROR' : newNumberOne.div(newNumberTwo);
+      break;
+    case '%':
+      result = (newNumberOne.div(100)).times(newNumberTwo);
+      break;
```

Comment:
`%` operator should turn the number into a percentage based number, `e.g. 95 % == .95`) which implies that it is a unary operator just like `+/-`.